### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by saketkum11**
Please review landing page and suggestion to better. 
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                <h1>WebKit-UI</h1>
+            </div>
+
+            <div class="m-lf">
+
+                <ul class="wrapper">
+                    <li class="list">
+                        <a href="" class="list-inline pd-3 text-color-0">Documentation</a>
+                    </li>
+
```
Comment (shraddha15-creator) [inline index.html]: Please remove empty lines and also format the code

**Review by shraddha15-creator — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (saketkum11) [reply index.html]: only the empty line

**Review by shraddha15-creator — APPROVED** [review]

**Review by shubhambishnoi — APPROVED** [review]

File: index.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+* {
+    box-sizing: border-box;
+    padding: 0rem;
+    margin: 0rem;
+}
+
+body {
+    font-family: 'Montserrat',
+        sans-serif;
+    font-family: 'Poppins',
```
Comment (znjs) [inline index.css:13]: The second `font-family` will overwrite the first one, you can remove either one

**Review by znjs — COMMENTED** [review]
check `index.css`

**Pull request merged**